index.html

```diff
@@ -57,13 +57,18 @@ <h1 class="display-3">Bikes for Refugees</h1>
 						<p class="lead">Providing donated bikes and accessories to refugees and asylum seekers in Scotland.</p>
 						<hr class="my-4">
 						<div class="buttons">
-							<a class="btn btn-primary btn-lrg" href="#">Donate a bike today</a>
+							<a class="btn  btn-orange" href="#">Donate a bike today</a>
 							<a class="btn btn-secondary btn-lrg" href="#">Volunteer</a>
 						</div>
 					</div>
+					<section class="flex-container row">
+						  <div class="flex-elem col-md-3">check availability</div>
+						  <div class="flex-elem col-md-3">donate bikes</div>
+						  <div class="flex-elem col-md-3">volunteer with us</div>
```

Comment:
Watch out here, you are overwriting Bootstrap classes (`row` and `col-md-3`) with your own. 
`.row` is already `display: flex`!
Your custom classes should not include any horizontal padding, margins or widths as this will clash with Bootstrap classes.